Support VTK 8.1

Trying to build with VTK 8.1.1:
```
BUILDSTDERR: CMake Error at CMakeLists.txt:58 (find_package):
BUILDSTDERR:   Could not find a configuration file for package "VTK" that is compatible
BUILDSTDERR:   with requested version "7.0".
BUILDSTDERR:   The following configuration files were considered but not accepted:
BUILDSTDERR:     /usr/lib64/cmake/vtk/VTKConfig.cmake, version: 8.1.1
BUILDSTDERR:     /lib64/cmake/vtk/VTKConfig.cmake, version: 8.1.1
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -55,7 +55,7 @@ find_package(Boost 1.53 REQUIRED COMPONENTS program_options filesystem system)
 # ToDo: remove support for VTK 6.1 at some point in future
 find_package(VTK 6.1 QUIET COMPONENTS vtkRenderingCore vtkImagingCore vtkFiltersCore vtkIOCore vtkIOGeometry vtkIOLegacy vtkIOXML vtkIOMINC vtkCommonCore vtkViewsCore vtkftgl vtksys vtkDICOMParser vtkexpat vtkzlib NO_MODULE)
 if(NOT VTK_FOUND)
-	find_package(VTK 7.0 REQUIRED COMPONENTS vtkRenderingCore vtkImagingCore vtkFiltersCore vtkIOCore vtkIOGeometry vtkIOLegacy vtkIOXML vtkIOMINC vtkCommonCore vtkViewsCore vtksys vtkDICOMParser vtkexpat vtkzlib NO_MODULE)
+	find_package(VTK REQUIRED COMPONENTS vtkRenderingCore vtkImagingCore vtkFiltersCore vtkIOCore vtkIOGeometry vtkIOLegacy vtkIOXML vtkIOMINC vtkCommonCore vtkViewsCore vtksys vtkDICOMParser vtkexpat vtkzlib NO_MODULE)
 endif()
 include(${VTK_USE_FILE})
 

